core/src/main/java/org/frankframework/stream/Message.java

```diff
@@ -206,7 +206,7 @@ private MessageNotClosedAction(String content) {
 		@Override
 		public void run() {
 			if (!calledByClose) {
-				LOG.info("Leak detection: Message was not closed properly! Content: [{}]", content);
+				LOG.info("Leak detection: Message was not closed properly! Content: [{}]", () -> StringUtils.substring(content, 0, 80));
```

Comment:
We bewaren deze `content` toch alleen voor deze logregel? Dan is het denk ik beter, om bij voorbaat al niet meer dan 80 characters ervan te bewaren.

Reply:
Ja inderdaad, pas ik aan, thanks!